lib/db.js

```diff
@@ -3,13 +3,30 @@ const levelup = require('level');
 const db = Promise.promisifyAll(levelup('./data'));
 
 exports.scores = {
-	get: function(id) {
-		return db.getAsync(id)
+	get: function(id, rev) {
+		return db.getAsync(id + ':' + rev)
 			.then(JSON.parse)
 	},
+	revisions: function(id) {
+		return db.getAsync(id + ':revisions')
+			.then(null, function (err) {
```

Comment:
LOL the `++revs` later automatically coerced the value of the promise to a Number, so I didn't notice this.

Great idea about JSON. Done.